packages/components/src/hooks/use-element-rect.ts

```diff
@@ -68,16 +50,35 @@ export const useElementDimension = (
         }
 
         return undefined;
-    }, [element, isVertical, startDim, watchSize]);
+    }, [element, isVertical, watchSize]);
 
-    return dimension;
+    return size;
 };
 
-const watchedSizeToDim = (isVertical: boolean, size: WatchedSize): number => {
-    return (isVertical ? size.height : size.width) || 0;
+const getElementDimensions = (element?: HTMLElement | null): ElementDimensions => {
+    if (element === null) {
+        return unMeasured;
+    }
+
+    if (element === undefined) {
```

Comment:
this is a bit weird to me since the function name is `getElementDimensions` but the element parameter is optional.